lib/util/util.c

```diff
@@ -481,13 +481,11 @@ static int bpf_mnt_check_target(const char *target)
 	struct stat sb = {};
```

Comment:
This should have given you an unused variable error.

Reply:
And indeed it does; guess maybe I forgot to compile-test this before pushing? Will fix